components/shooter.py

```diff
@@ -71,6 +77,7 @@ def __init__(self) -> None:
         flywheel_config.apply(flywheel_motor_config)
         flywheel_config.apply(flywheel_pid)
         flywheel_config.apply(flywheel_gear_ratio)
```

Comment:
I think we probably want to set the same configs on both motors? In particular the motor output config